doc/wiki.txt

```diff
@@ -616,6 +617,9 @@ OPTIONS                                                   *wiki-config-options*
   visual mode, respectively. The available `<plug>` mappings are listed in
   |wiki-mappings|.
 
+  For any local mappings that should only be applied in journal entries,
+  consider setting |g:wiki_mappings_journal_local|.
+
```

Comment:
Perhaps one should add an example for this as well?

Reply:
Added an example and an extra table for the defaults.